include/mbedtls/error.h

```diff
@@ -80,6 +80,7 @@
  * CHACHA20  3                  0x0051-0x0055
  * POLY1305  3                  0x0057-0x005B
  * CHACHAPOLY 2 0x0054-0x0056
+ * PLATFORM  1  0x0080-0x0080
```

Comment:
The low level error codes are limited up to `0x7f`
This causes `mbedtls_strerror()` not to recognize `0x80, as it is as a low level error code, but the value belongs to a high level error code.
The valuse should be changed, but it is already merged in [Mbed OS](https://github.com/ARMmbed/mbed-os/blob/master/features/mbedtls/platform/inc/platform_mbed.h#L28)
@sbutcher-arm  Should we change the value, causing possible ABI issues if users have specifically expect this error code or should we change the limit of the low level error code?

Reply:
@RonEld - Can you please change the line to RESERVED? The value isn't properly allocated, so can be reused at some point in the future. It never touched the Mbed TLS standalone release.

Reply:
Actually, I should change this description to 0x0070
What do you want me to change to reserved? Since it hasn't been merged yet to Mbed TLS, why should we reserve this value?